og_ui/og_ui.routing.yml

```diff
@@ -16,6 +16,23 @@ og_ui.settings:
   requirements:
     _permission: 'administer group'
 
+og_ui.subscribe:
+  path: 'group/{entity_type_id}/{entity_id}/subscribe/{field_name}'
+  defaults:
+    _controller: '\Drupal\og_ui\Controller\SubscriptionController::subscribe'
+    _title: 'Join Group'
```

Comment:
This will need to be a callback, as it will be 'Subscribe to group' or 'Request subscription to group` -- according to the group's permissions.